### PR TITLE
Miguel/2506 cant open links on validator pages

### LINKED ISSUE #2506: Can't open links on validator pages
**Describe the bug**

"Under validators website url does not link externally or with target _blank" (from Appzi)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: 
 - Browser: 
 - Version: 

**Smartphone (please complete the following information):**
 - Device: 
 - OS: 
 - Browser 
 - Version 

**Additional context**

### PR DESCRIPTION
**Pull request by migueog**
Closes #2506 

**Description:**

- Prepends `https://` to links that do not include `https://` or `http://`

- Added `test:updateSnapshot` to `package.json` scripts 

Thank you! 🚀

---

For contributor:

- [x] Added changes entries. Run `yarn changelog` for a guided process.
- [x] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "lint:fix": "eslint --fix -f ./node_modules/eslint-friendly-formatter \"{,**/}*.{js,vue}\" && stylelint \"**/*.{css,vue}\" --fix",
     "test": "yarn run lint && yarn test:unit && yarn test:e2e",
     "test:unit": "cross-env NODE_ENV=testing jest --maxWorkers=2",
+    "test:updateSnapshot": "cross-env NODE_ENV=testing jest --updateSnapshot",
```
Comment (faboweb) [inline package.json]: you should probably do this on a file by file basis. `jest PageValidator -u`

**Comment by faboweb** [comment]
Loos good, small comment

File: test/unit/specs/components/staking/PageValidator.spec.js [hunk 2]
```diff
@@ -207,8 +207,22 @@ describe(`PageValidator`, () => {
...
+      expect(wrapper.vm.website).toBe(`--`)
+    })
+
+    it(`shows https website url`, () => {
+      validator.description.website = `www.monty.ca`
+      wrapper.setData({ validator })
+      expect(wrapper.vm.website).toBe(`https://www.monty.ca`)
+    })
+
+    it(`shows http website url`, () => {
```
Comment (jbibla) [inline test/unit/specs/components/staking/PageValidator.spec.js]: i would add one more test that checks to make sure nothing happens if it already has `https`

File: package.json (hunk 1 above)
Comment (faboweb) [reply package.json]: so this should probably removed to not confuse developers to use it.

Comment (migueog) [reply package.json]: OK sounds good. Thanks!

File: test/unit/specs/components/staking/PageValidator.spec.js (hunk 2 above)
Comment (migueog) [reply test/unit/specs/components/staking/PageValidator.spec.js]: Good call. Thanks for the suggestion.

**Comment by jbibla** [comment]
nice work @migueog  🎉

welcome aboard! 🚀

**Pull request merged**